CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Automatically add labels on Kubernetes Nodes to LINSTOR satellites as Auxiliary Properties. This enables using
   Kubernetes labels for volume scheduling, for example using `replicasOnSame: topology.kubernetes.io/zone`.
 
+### Changed
+
+- Enable CSI topology by default, allowing better volume scheduling with `volumeBindingMode: WaitOnFirstConsumer`.
```

Comment:
In [the docs](https://kubernetes.io/docs/concepts/storage/storage-classes/) I only find `WaitForFirstConsumer`.

Reply:
good catch